### DISCUSSION TITLE
uuid python requirement causing issues

### DISCUSSION THREAD

**Discussion by ajayvignesh01** [vercel/community#3406] [Help]
### Summary

I added uuid to 'requirements.txt' and also as an import statement at the top of my py file. I tested all the functions in the python file locally, and it runs without errors. When I deploy to vercel however, I get the following error in logs:

```
[ERROR] Runtime.UserCodeSyntaxError: Syntax error in module 'vc__handler__python': invalid syntax (uuid.py, line 138)
Traceback (most recent call last):
  File "/var/task/uuid.py" Line 138
                if not 0 <= time_low < 1<<32L:
```

This prevents any of the other python functions from working as well even though they don't require uuid. For the time being, I have replaced uuid with random, but this is not a viable solution for my website long-term.

Side note: python function size is abnormally high at around 45mb. Tried some of the solutions at [https://github.com/orgs/vercel/discussions/103](url), but none of them helped.

### Steps to Reproduce

```
from http.server import BaseHTTPRequestHandler
import uuid
 
class handler(BaseHTTPRequestHandler):
 
    def do_GET(self):
        self.send_response(200)
        self.send_header('Content-type','text/plain')
        self.end_headers()
        self.wfile.write(str(uuid.uuid4()).encode('utf-8'))
        return
```

and then also add 'uuid' to requirement.txt

**Comment by dyagee** [comment]
I encountered a similar problem,

**Comment by scottoasis** [comment]
We encounter this issue as well. As it turned out, the `uuid` package is now built-in to python. Removing the `uuid` from `requirements.txt` or `Pipfile` should fix the issue, and there is no need to alter any code.

Source of reference: https://stackoverflow.com/a/33613139

**Reply by dyagee** [reply]
I later used  os.urandom to improvise and generate random bytes.

But I will give this a try.
